lib/MusicBrainz/Server/Entity/Genre.pm

```diff
@@ -0,0 +1,42 @@
+package MusicBrainz::Server::Entity::Genre;
+
+use Moose;
+use MusicBrainz::Server::Entity::Types;
+
+extends 'MusicBrainz::Server::Entity::CoreEntity';
+with 'MusicBrainz::Server::Entity::Role::Comment';
+with 'MusicBrainz::Server::Entity::Role::LastUpdate';
+
+sub entity_type { 'genre' }
+
+around TO_JSON => sub {
+    my ($orig, $self) = @_;
+
+    return {
+        %{ $self->$orig },
```

Comment:
We can probably remove this method since it doesn't do anything and isn't likely to have any additions in the near future (but maybe I'm forgetting something).